src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -79,57 +80,43 @@ import { DatePicker } from '@orfium/ictinus';
 
 <Props of={DatePicker} />
 
-# Calendar
+# DatePicker
```

Comment:
by removing calendar is not feasible to check in chromatic now the changes, we can discuss this

Reply:
fixed on latest commit